dynomitemanager-core/src/main/java/com/netflix/dynomitemanager/config/FloridaConfig.java

```diff
@@ -293,7 +303,16 @@
      */
     @DefaultValue("16")
     @PropertyName(name = "dyno.ardb.rocksdb.maxwritebuffernumber")
-    public int getArdbRocksDBMaxWriteBufferNumber();
+    public int getRocksDBMaxWriteBufferNumber();
+
+    /**
+     * Loglevel
+     * 
+     * @return the loglevel to set for RocksDB
+     */
+    @DefaultValue("warn")
+    @PropertyName(name = "dyno.ardb.rocksdb.loglevel")
+    public String getArdbLoglevel();
```

Comment:
getArdbRocksDBLoglevel??
make the function name and the FP name consistent.

Should the FP name have rocksdb in it?

Reply:
This is an ARDB configuration (hold for all storage engines). I am trying to differentiate between RocksDB and ARDB generic configurations.

Reply:
why do we have rocksdb here in the FP name then?

Reply:
You are right, I fixed it.